crates/bevy_pbr/src/render/light.rs

```diff
@@ -968,10 +962,7 @@ pub fn prepare_lights(
             // index to shadow map index, we need to subtract point light count and add directional shadowmap count.
             spot_light_shadowmap_offset: num_directional_cascades_enabled as i32
                 - point_light_count as i32,
-            environment_map_smallest_specular_mip_level: environment_map
-                .and_then(|env_map| images.get(&env_map.specular_map))
-                .map(|specular_map| specular_map.mip_level_count - 1)
-                .unwrap_or(0),
+            environment_map_smallest_specular_mip_level: 0, /* FIXME */
```

Comment:
This needs to be fixed. Specular envmaps won't look correct at all. What prevented you from doing so?

Reply:
So I ended up just using `textureNumLevels()` as the comment in `environment_map.wgsl` suggests. Since reflection probes are disabled on WebGL 2, we don't have to worry about compatibility for those. I did need to add a `smallest_specular_mip_level_for_view` field to the `LightProbes` struct though, since that one *is* used on WebGL 2.